demisto_sdk/commands/generate_unit_tests/generate_unit_tests.py

```diff
@@ -91,21 +91,28 @@ def create_command_to_generate_dict(self):
         """
         Parses command_examples into dictionary of command name and arguments.
         """
+        display_commands = []
         for command in self.command_examples:
             command_line = command.split(' ')
             command_name = self.command_name_transformer(command_line[0])
+            command_name_without_vendor = command_name.split('_', 1)[1]
             command_dict = {}
+            display_commands.append(command_name)
             for arg in command_line[1:]:
                 key = arg.split('=')[0]
                 value = arg.split('=')[1]
                 value = value.replace('"', '')
                 command_dict.update({key: value})
             if command_name in self.commands_to_generate:
                 self.commands_to_generate.get(command_name).append(command_dict)
+            elif command_name_without_vendor in self.commands_to_generate:
+                self.commands_to_generate.get(command_name_without_vendor).append(command_dict)
```

Comment:
not sure what you mean, but the logic is similar to line 106 (get(command_name)), which works. the get is used only if the command name is in the commands to generate list, in that case it will return a reference which we will update.

At first show of the command name, we will create the entry.